parm/wave/bull_awips_gfswave.uglo_15km

```diff
@@ -14,7 +14,7 @@ export b46077=AGGA48_KWBJ_OSBM12
 export b46079=AGGA48_KWBJ_OSBM13
 export b46105=AGGA48_KWBJ_OSBM14
 export b46107=AGGA48_KWBJ_OSBM15
-export b46265=AGGA48_KWBJ_OSBM16
+#####export b46265=AGGA48_KWBJ_OSBM16
```

Comment:
Will these be uncommented down the road?

Reply:
For future versions, hopefully some of these will be commentated yes.

Reply:
Gotcha, thanks!